libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1451,6 +1451,61 @@ string YulUtilFunctions::nextArrayElementFunction(ArrayType const& _type)
 	});
 }
 
+string YulUtilFunctions::copyArrayFromStorageToMemoryFunction(ArrayType const& _from, ArrayType const& _to)
+{
+	solAssert(_from.baseType() == _to.baseType(), "");
+	solAssert(_from.isDynamicallySized() == _to.isDynamicallySized(), "");
+	if (!_from.isDynamicallySized())
+		solAssert(_from.length() == _to.length(), "");
```

Comment:
Is it possible to copy from storage to memory if `_to.length() > _from.length()`? I think it should be allowed

Reply:
The type checker does not allow it currently.